Sources/Clappr_iOS/Classes/Base/Player.swift

```diff
@@ -97,6 +97,15 @@ open class Player: BaseObject {
         core?.on(Event.didChangeActivePlayback.rawValue) { [weak self] _ in self?.bindPlaybackEvents() }
         core?.on(InternalEvent.userRequestEnterInFullscreen.rawValue) { [weak self] (info: EventUserInfo) in self?.trigger(Event.requestFullscreen.rawValue, userInfo: info) }
         core?.on(InternalEvent.userRequestExitFullscreen.rawValue) { [weak self] (info: EventUserInfo) in self?.trigger(Event.exitFullscreen.rawValue, userInfo: info) }
+        bindMediaControlEvents()
```

Comment:
I'd rather move it between line `84` and line `85`, because it looks like `bindMediaControlEvents` is now a side effect of calling `bindCoreEvents`. We know that both are core dependant, but semantically, they are different "layers".

Reply:
Yeah, it makes perfect sense.